gadget-container/cleanup.sh

```diff
@@ -22,8 +22,12 @@ for i in ocihookgadget prestart.sh poststop.sh ; do
 done
 
 # CRIO hooks
-rm -f /host/etc/containers/oci/hooks.d/gadget-prestart.json
-rm -f /host/etc/containers/oci/hooks.d/gadget-poststop.json
+for HOOK_PATH in "/host/etc/containers/oci/hooks.d" \
+                 "/host/usr/share/containers/oci/hooks.d/"
+do
+  rm -f $HOOK_PATH/gadget-prestart.json
+  rm -f $HOOK_PATH/gadget-poststop.json
+done
```

Comment:
Why is this showing these changes here? Probably you have to rebase again?

Reply:
Yep, another rebase fixed that